ui/packages/atlasmap/src/Views/ColumnMapperView/Actions/EnableJavaClassAction.tsx

```diff
@@ -4,10 +4,12 @@ import { AddCircleOIcon } from "@patternfly/react-icons";
 
 export interface IEnableJavaClassActionProps {
   onCustomClassSearch: () => void;
+  id: string;
```

Comment:
A pattern I found that works is instead of adding `id`s like you can simply pass the `data-testid` to the component and then pass all the "unknown" props to the root element. Example in the following lines